suite/tests/CMakeLists.txt

```diff
@@ -3331,6 +3353,12 @@ if (CLIENT_INTERFACE)
       torunonly_drcacheoff(instr-only-trace ${ci_shared_app} "-instr_only_trace" "" "")
       torunonly_drcacheoff(filter-and-instr-only-trace ${ci_shared_app} "-instr_only_trace -L0_filter" "" "")
 
+      # __builtin_prefetch used in the test is available only on gcc.
```

Comment:
Repeating comment from before: no, it is not limited to gcc.  It is supported by clang as well.

Reply:
Oops. Sorry I didn't address this before. __builtin_prefetch is not available in MSVC. I changed the condition to reflect that.